### DISCUSSION TITLE
Is this free software?

### DISCUSSION THREAD

**Discussion by quantitative-technologies** [imapsync/imapsync#481] [General]
Generally I thought software in a github repo can be used free of charge, but your upstream website says free below 3GB.

Is there a 3GB limitation when using the software in this repo?

**Comment by gilleslamiral** [comment]
> Generally I thought software in a github repo can be used free of charge, but your upstream website says free below 3GB.
> 

The online service is not imapsync, it doesn't require any download. It's an online service, with no installation.

> Is there a 3GB limitation when using the software in this repo?

There is no limitation here.
Neither at the upstream website by the way.
The upstream website says it's gratis under 3G but it is also gratis above 3G, there is no limit. 
Have a try and see.
I restored the warning about 3G because the sales are cut by half without it.
It's a fake warning. To pay the bills and continue to work on imapsync.

**Reply by quantitative-technologies** [reply]
Haha, got it!

Thanks for this.
